server/sonar-server/src/main/java/org/sonar/server/computation/snapshot/Snapshot.java

```diff
@@ -47,27 +54,35 @@ public long getCreatedAt() {
     @CheckForNull
     private Long id;
     @CheckForNull
+    private String uuid;
```

Comment:
it can be `null` in the Builder and it is when the Builder is instanciated
non nullity is only enforced in the bean